docs/README.md

```diff
@@ -84,13 +84,14 @@ The `lxd_remote` block supports:
 
 * `address` - *Optional* - The address of the LXD remote.
 
-* `default` - *Optional* - Whether this should be the default remote. This
-	remote will then be used when one is not specified in a resource. Valid
-	values are `true` and `false`.
-  If you choose to _not_ set default=true on any `lxd_remote` this provider
-  will attempt to connect to an LXD server running on the same host through
-  the UNIX socket. See `Undefined Remote` for more information
-  The default can also be set with the `LXD_REMOTE` Environment variable.
+* `default` - *Optional* - Whether this should be the default remote. 
+	This remote will then be used when one is not specified in a resource.
+	Valid values are `true` and `false`.
+	If you choose to _not_ set default=true on an `lxd_remote and do not specify
```

Comment:
Small syntax error. Missing the closing backtick on `lxd_remote` 😄

Reply:
Oops, sorry should be fixed now. :)